### PR TITLE
Rename 'Stable' release to 'Current'

### PR DESCRIPTION
**Pull request by fhemberger**
PR for #669:
- I haven't changed any release blog posts _prior 4.0_. The old Node.js version numbers used the terms stable/unstable.
- We will need a redirect from `/<lang>/downloads/stable` to `/<lang>/downloads/current`: https://github.com/nodejs/build/pull/399
- ~~Changing markdown files while running `npm run serve` crashes the process:~~ Fixed (cherry-picked to `master`)
- ~~Some tests are failing for release blogpost generation, @phillipj can you please take a look?~~

### CODE REVIEW

**Comment by MylesBorins** [comment]
@fhemberger thanks for getting this started. I'll take a proper look tomorrow

**Comment by lpinca** [comment]
> Changing markdown files while running npm run serve crashes the process:

I think this is unrelated to this change, I sporadically received the same error in the last days.

File: locale/it/index.md [hunk 1]
```diff
@@ -6,10 +6,10 @@ labels:
   download-for: Download per
   other-downloads: Altri Downloads
   other-lts-downloads: Altri Download LTS
-  other-stable-downloads: Altri Download Stabili
-  stable: Stabile
+  other-current-downloads: Altri Download Stabili
+  current: Stabile
```
Comment (lpinca) [inline locale/it/index.md]: `current: Corrente`

File: locale/it/index.md [hunk 2]
```diff
@@ -6,10 +6,10 @@ labels:
   download-for: Download per
   other-downloads: Altri Downloads
   other-lts-downloads: Altri Download LTS
-  other-stable-downloads: Altri Download Stabili
-  stable: Stabile
+  other-current-downloads: Altri Download Stabili
+  current: Corrente
```
Comment (fhemberger) [inline locale/it/index.md:10]: @lpinca Grazie! ;)

Comment (MylesBorins) [inline locale/it/index.md:10]: this is the exact kind of stuff I knew I would miss. Thanks for getting on it so quickly @fhemberger !!!

File: locale/it/index.md [hunk 3]
```diff
@@ -6,10 +6,10 @@ labels:
   download-for: Download per
   other-downloads: Altri Downloads
   other-lts-downloads: Altri Download LTS
-  other-stable-downloads: Altri Download Stabili
-  stable: Stabile
+  other-current-downloads: Altri Download Stabili
```
Comment (lpinca) [inline locale/it/index.md:9]: "Altri download della versione corrente" or "Altri download correnti".
Not sure about casing.

cc: @parro-it @piccoloaiutante

P.S. we can do these things in another pr :)

Comment (parro-it) [inline locale/it/index.md:9]: "Altri download della versione corrente" seems better to me.

**Comment by hiroppy** [comment]
@fhemberger @lpinca I think `buildLocale` shouldn't get `source` as `string` when file is changed. 

https://github.com/nodejs/nodejs.org/blob/master/server.js#L44
https://github.com/nodejs/nodejs.org/blob/master/build.js#L57

File: tests/scripts/release-post.test.js [hunk 4]
```diff
@@ -194,13 +194,13 @@ test('fetchVersionPolicy(<version>)', (t) => {
...
 
-  t.test('finds "Stable" version policy', (t) => {
+  t.test('finds "Current" version policy', (t) => {
     const github = nock('https://raw.githubusercontent.com')
       .get('/nodejs/node/v4.1.0/CHANGELOG.md')
       .replyWithFile(200, changelogFixture)
 
     releasePost.fetchVersionPolicy('4.1.0').then((policy) => {
-      t.equal(policy, 'Stable')
+      t.equal(policy, 'Current')
```
Comment (lpinca) [inline tests/scripts/release-post.test.js:203]: This assertion fails because the channel is resolved as "Stable" from the [fixture](https://github.com/nodejs/nodejs.org/blob/04b4e0473c73b8377c3aaa9b2c0a9b2e907afedc/tests/scripts/CHANGELOG.fixture.md).
Maybe it makes sense to create a new fixture faking a changelog for the "Current" channel.

**Comment by lpinca** [comment]
@abouthiroppy it makes sense.

**Comment by fhemberger** [comment]
@abouthiroppy Good catch, thanks! Crash is fixed.

**Comment by jasnell** [comment]
@node/ctc ... FYI
The naming change LGTM.

File: tests/scripts/release-post.test.js (hunk 4 above)
Comment (phillipj) [inline tests/scripts/release-post.test.js:203]: Does the name change affect existing releases? If not, this assertion should not be changed as v4.1.0 should still be considered "stable".

Comment (stevemao) [inline tests/scripts/release-post.test.js:203]: > Does the name change affect existing releases?

I vote yes.

Comment (phillipj) [inline tests/scripts/release-post.test.js:203]: > I vote yes.

I would agree in the eyes of consistency, but that would possibly mean updating all previous changelogs/blogs etc, I'm not sure that's preferable.

@fhemberger as I'm not sure what to do about previous releases, I've made two branches with contains fixes for both scenarios:
- `rename-stable-old-still-stable`: old releases should still be named "Stable"
- `rename-stable-incl-old-releases`: rename old releases to "Stable" as well

Merge or cherry-pick what ever commit you see fit. IMO keeping "Stable" for old releases would be okey for this PR, further discussion and a permanent fix could be raised in a separate issue/PR.

File: locale/ko/index.md [hunk 5]
```diff
@@ -6,10 +6,10 @@ labels:
   download-for: Download for
   other-downloads: 다른 운영 체제
   other-lts-downloads: 다른 LTS 다운로드
-  other-stable-downloads: 다른 안정 버전 다운로드
```
Comment (preco21) [inline locale/ko/index.md:9]: @fhemberger Could you change '안정' to '현재' on here? ('stable' to 'current' in Korean)
Any other things are LGTM.

Comment (fhemberger) [inline locale/ko/index.md:9]: @preco21 Updated the translation, also on the next line.

**Comment by fhemberger** [comment]
@phillipj I went with the old releases to be changed as well. Updated release posts and weekly updates for v4.x/5.x releases.

Anything else missing or are we good?

File: locale/ko/index.md (hunk 5 above)
Comment (preco21) [inline locale/ko/index.md:9]: @fhemberger Cool! Thank you.

File: locale/en/blog/release/v4.1.1.md [hunk 6]
```diff
@@ -2,7 +2,7 @@
 date: 2015-09-23T02:01:38.545Z
 version: 4.1.1
 category: release
-title: Node v4.1.1 (Stable)
+title: Node v4.1.1 (Current)
```
Comment (MylesBorins) [inline locale/en/blog/release/v4.1.1.md:5]: I'm not under the impression that we will be changing the release type of old releases... these posts should likely remain unchanged

Comment (MylesBorins) [inline locale/en/blog/release/v4.1.1.md:5]: just seeing that this was discussed above. I for one would push towards not making a decision in this PR and supporting the old names. There is no way for us to update release commits, and I do not believe there is any intention of modifying the changelogs. It would be odd if the release posts + changelogs are not the same

Comment (fhemberger) [inline locale/en/blog/release/v4.1.1.md:5]: That's why I did this in a separate commit, so the others can be cherry-picked, if needed.

**Comment by mikeal** [comment]
I'm going to merge this. We have other messaging lined up around this terminology going out today and the website should line up with it. So far there have been zero objections from any member of any WG.

**Comment by fhemberger** [comment]
@mikeal We need this redirect to land as well, otherwise the old download links for stable builds will be broken: https://github.com/nodejs/build/pull/399

**Comment by ghost** [comment]
also: should we note this as news and post about it? just to archive it officially

**Comment by jasnell** [comment]
I will be including a note about the change in the v6 release notes.

**Comment by MylesBorins** [comment]
so this was landed with change to all of the past blog posts from stable -> current. We might want to revisit this change as it is going to create a delta from the changelog and releases where this is not going to change
